spec/lib/container_orchestrator_spec.rb

```diff
@@ -317,6 +317,30 @@
           subject.watch_pods(100)
         end
       end
+
+      it "#get_pod_by_namespace_and_hostname" do
+        pods = [
+          double("pod", :metadata => double("pod-metadata", :name => "ui-worker.test.svc.cluster.local")),
```

Comment:
For accuracy, the name should be something like `3r1-ui-d57f497f6-qc9s6`

Reply:
perfect..I'll update

Reply:
Update with more realistic looking hostnames